c2corg_ui/static/js/documentediting.js

```diff
@@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
     scope: true,
     controller: 'appDocumentEditingController',
     controllerAs: 'editCtrl',
-    bindToController: true
+    bindToController: true,
+    link: function(scope, el, attrs, ctrl) {
+      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
```

Comment:
Good idea, I can try that.
But I'm not really sure to understand what is the problem with waiting 500ms to evaluate this max_steps (which isn't immediately needed neither) ? The DOM has loaded and the user can start editing/creating a document, this variable will just appear a 0.5s later.